src/com/tsm/courses/dmytro231/lessons4/MassNn.java

```diff
@@ -0,0 +1,54 @@
+package com.tsm.courses.dmytro231.lessons4;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+
+public class MassNn {
+    public static void main(String[] args) {
+
+        Scanner input = new Scanner(System.in);
+        System.out.println("Enter the array size: ");
+        if (input.hasNextInt()) {
+            int t = input.nextInt();
+            if (t >= 5) {
+                if (t <= 10) {
+                    int[] mas1 = new int[t];
```

Comment:
Better to separate logically code, example: 1) Input size (with checks) 2) perform all calculations after - no need to place calculation logic inside input checks